agent/providers/lib/v8.go

```diff
@@ -1250,6 +1216,22 @@ func (v *V8) newObjectStoreObjectTemplate() *v8.ObjectTemplate {
 	return objectStore
 }
 
+func (v *V8) marshalValue(val *v8.Value) ([]byte, error) {
+	if val.IsObject() || val.IsArray() {
+		return val.MarshalJSON()
+	} else if val.IsBigInt() {
```

Comment:
Will this cause problems when/if endianness changes?

Reply:
The `IsBigInt` check itself shouldn't cause problems if endianness changes. What we do with the bytes when they are unmarshaled at a later time is something to consider. We are using a string representation with no type hint inside key/value and object stores 🤔